### ISSUE TITLE
roachtest: flake in acceptance/gossip/restart-node-one

### ISSUE THREAD

**Issue by knz** [cockroachdb/cockroach#106834]
Found here: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/10915134?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildTestsSection=true
```
(gossip.go:441).runGossipRestartNodeOne: output in run_154221.366247435_n1_cockroach-start-inse:  ./cockroach start --insecure --background --store={store-dir} --log-dir={log-dir} --cache=10% --max-sql-memory=10% --listen-addr=:$[{pgport:1}+10000] --http-port=$[{pgport:1}+1] --join={pghost:1}:{pgport:1} --advertise-addr={pghost:1}:$[{pgport:1}+10000] > {log-dir}/cockroach.stdout 2> {log-dir}/cockroach.stderr returned: COMMAND_PROBLEM: exit status 1
(test_runner.go:1122).func1: 4 dead node(s) detected
test artifacts and logs in: /artifacts/acceptance/gossip/restart-node-one/run_1
(gossip.go:441).runGossipRestartNodeOne: output in run_154221.366247435_n1_cockroach-start-inse:  ./cockroach start --insecure --background --store={store-dir} --log-dir={log-dir} --cache=10% --max-sql-memory=10% --listen-addr=:$[{pgport:1}+10000] --http-port=$[{pgport:1}+1] --join={pghost:1}:{pgport:1} --advertise-addr={pghost:1}:$[{pgport:1}+10000] > {log-dir}/cockroach.stdout 2> {log-dir}/cockroach.stderr returned: COMMAND_PROBLEM: exit status 1
(test_runner.go:1122).func1: 4 dead node(s) detected
```

Jira issue: CRDB-29734

**Comment by shralex** [comment]
@tbg do you have time to take a look ?

**Comment by nicktrav** [comment]
Leaving this one with KV. If it looks like a Repl issue, please update the T- label to reflect.

**Comment by jayshrivastava** [comment]
👋🏽 I just hit this in 23.1. Looks like we need to backport https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/11798127?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true

**Comment by rafiss** [comment]
also failed at https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/11859058?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildTestsSection=true

**Comment by kvoli** [comment]
23.1 backport PR #111265

**Comment by kvoli** [comment]
Fixed by backport #111265.
